src/keys.ts

```diff
@@ -54,15 +55,27 @@ function parseExtractedKey(key: ExtractedKey, config: Config): TranslationKey {
     const nsSeparatorPos = cleanKey.indexOf(config.nsSeparator);
 
     if (nsSeparatorPos !== -1) {
+      if (key.keyPrefix) {
+        throw new Error(
```

Comment:
hat is i18next behavior when you use both? Does it error out or pick one of the two namespaces? I think we should align with their behavior.

In any case, we need a test for this.

Reply:
I'm not really sure what will happend if i use both, they just have warning in documentation it won't work. 
https://www.i18next.com/overview/api#getfixedt

I'll test exact behaviour of i18next and try to make tests for it